tests/shared_disk/test_shared_disk_windows_migration.py

```
@@ -0,0 +1,200 @@
+@pytest.mark.usefixtures("cleanup_migrated_vms")
+class TestSharedDiskWindowsMigration:
+    """Cold migrate Windows VMs with VM-level migrateSharedDisks overrides."""
+
+    storage_map: StorageMap
+    network_map: NetworkMap
+    plan_resource: Plan
+
+    def test_label_shared_disk(
+        self,
+        prepared_plan: dict[str, Any],
+        source_provider: "BaseProvider",
```

Ruff (0.15.18)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)